wahoomc/main.py

```diff
@@ -103,5 +107,7 @@ def run(run_level):
         if o_input_data.save_cruiser is True:
             o_osm_maps.make_and_zip_files('.map', o_input_data.zip_folder)
 
+        log.info('Total time %.5f, %.5f', time.process_time()-process_time, time.perf_counter()-wall_clock)
```

Comment:
I tried to have a consistent logging by using `#` for the "headline" and `+` for the "items" of the process. Here there should be a `#` in my opinion.